lib/testing/src/lib/core/utils/browser-actions.ts

```diff
@@ -21,9 +21,17 @@ import { BrowserVisibility } from '../utils/browser-visibility';
 export class BrowserActions {
 
     static async click(elementFinder: ElementFinder): Promise<void> {
-        await BrowserVisibility.waitUntilElementIsVisible(elementFinder);
-        await BrowserVisibility.waitUntilElementIsClickable(elementFinder);
-        await elementFinder.click();
+        try {
+            await BrowserVisibility.waitUntilElementIsClickable(elementFinder);
+            await elementFinder.click();
+        } catch (clickErr) {
+          try {
+            await browser.executeScript(`arguments[0].scrollIntoView();`, elementFinder);
```

Comment:
This makes the clickExecuteScript method from below not needed anymore

Reply:
It does, but I cannot remove it, since it might affect the existing tests.
This is not a refactoring PR :)